released/packages/coq-haskell/coq-haskell.1.0.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+
+homepage: "https://github.com/jwiegley/coq-haskell"
+dev-repo: "git+https://github.com/jwiegley/coq-haskell.git"
+bug-reports: "https://github.com/jwiegley/coq-haskell/issues"
+license: "BSD-3-Clause"
+
+synopsis: "A library to provide Haskell-familiar constructions in Coq"
+description: """
+This library is designed for Haskell users who are either using Coq to build
+code intended for extraction to Haskell, or who wish to prototype/prove their
+algorithms in Coq. It provides a collection of definitions and notations to
+make Gallina more familiar to Haskellers.
+"""
+
+build: [make "-j%{jobs}%"]
```

Comment:
```suggestion
build: [make]
```
According to CI, the Makefile has a serious problem with parallelism, which leads to nondeterministic failures. The only approach I see to fix this is by disabling parallelism completely.